os/arch/arm/src/imxrt/imxrt_mtd_flash.c

```diff
@@ -279,28 +277,22 @@ static inline int imxrt_bulkerase(struct imxrt_dev_s *priv, unsigned long arg)
 	/* Deselect the FLASH */
 
 	SPI_SELECT(priv->dev, SPIDEV_FLASH, false);
+#endif
 	
-	size_t sectors_erased = 0;
-	size_t startsector = 8;
-//	fdbg("startsector: %d nsectors: %d\n", (long)startsector, (int)priv->nsectors);
-	fdbg("startsector: %d nsectors: %d\n", (long)startsector, 2);
+	off_t sectors_erased = 0;
```

Comment:
This points that current target sector, renaming would be nice.

Reply:
Strictly speaking, imxrt_bulkerase is not used just to run FS TCs successfully.
So I reverted this function, this will be implemented in another PR.